docs/extensibility/snippets/MailDevResource/global.json

```diff
@@ -0,0 +1,3 @@
+{
+    "sdk": { "version": "8.0.303" }
```

Comment:
Do you want a rollfoward strategy here so it doesn't fail if you don't have exactly 8.0.303 installed?

Reply:
I don't know if I want a _global.json_ file at all. I just know that since I have .NET 9 preview installed, .NET Aspire doesn't work at all without it.